src/main/java/com/rocketden/main/service/GameManagementService.java

```diff
@@ -95,6 +99,33 @@ public RoomDto startGame(String roomId, StartGameRequest request) {
         return roomDto;
     }
 
+    public RoomDto playAgain(String roomId, PlayAgainRequest request) {
```

Comment:
At some point, we should remove the `game` from `currentGameMap`. I thought maybe this would be done after the game ends, or maybe it could be done after a set amount of time or when all the users leave. Thoughts?

Reply:
Yeah sounds good - I'll wait for the other PR to be merged first, that way I can pull those changes and add the calls to a new endGame method at the correct places.

Reply:
Actually, this might present some issues - the play again method requires the game to still exist by setting the play again variable and sending a game socket message, so if games were removed, it likely couldn't be immediately after the game ends. Thoughts? We could do delete after a certain amount of time, although tbh that doesn't feel desirable to be too reliant on timers and stuff

Reply:
Maybe we could discuss different ideas on a call, but there could be a countdown for rejoining the game (visible to the users, and if the game is not restarted, then you can't restart it?), or a new Game could be created every time (we'd have to figure out another way to restart, perhaps by creating a different subscription especially for the "restart and create new game" purpose). You could potentially make a new "create room" method that allows you to specify the room ID by making it the previous one, and the game can be stored in the database under a different ID (to prevent duplicates, we'd likely want to do this anyway).

Reply:
That's kind of just a braindump of ideas, we can talk through this more another time.

Reply:
Sounds good - given that this might also be dependent on what we do with the results page, I added another card for it in the backlog with the thought that it could be dealt with in its own PR (in addition to all the saving game info to the database stuff which could be relevant to an endGame method)